src/main/java/DataManager.java

```diff
@@ -0,0 +1,71 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import commands.Task;
+import commands.Deadline;
+import commands.Event;
+import commands.Todo;
+
+public class DataManager {
+
+    private static void writeToFile(String filePath, String data) throws IOException {
+        Path path = Paths.get(filePath);
+        Files.write(path, data.getBytes());
+    }
+
+    public static void saveData(List<Task> list) throws IOException {
+        StringBuilder stringBuilder = new StringBuilder();
+        for (Task t : list) {
+            stringBuilder.append(t.toString()).append("\n");
+        }
+        writeToFile("src/main/java/data.txt", stringBuilder.toString());
+    }
+
+    public static List<Task> readData(String filePath) throws IOException{
+        List<Task> list = new ArrayList<>();
+        Path path = Paths.get(filePath);
+        if (Files.exists(path)) {
+            List<String> lines = Files.readAllLines(path);
+            for (String line : lines) {
+                if (!line.equals("")) {
+                    list.add(parseLine(line));
+                }
+            }
+            return list;
+        } else {
+            return list;
+        }
+    }
+
+    private static Task parseLine(String str) {
+        String brackets = str.substring(0, 7);
```

Comment:
Avoid using magic number, can give a meaningful variable name to 0 and 7 for easier understanding of why the value 0 and 7 is chosen particularly,